docs/content.zh/docs/connectors/pipeline-connectors/elasticsearch.md

```diff
@@ -0,0 +1,275 @@
+      </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>username</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>用于连接 Elasticsearch 实例认证的用户名。</td>
+    </tr>
+    <tr>
+      <td>password</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>用于连接 Elasticsearch 实例认证的密码。</td>
+    </tr>
+    <tr>
+      <td>batch.size.max</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">500</td>
+      <td>Integer</td>
+      <td>每个批量请求的最大缓冲操作数。 可以设置为'0'来禁用它。</td>
+    </tr>
+    <tr>
+      <td>inflight.requests.max</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">5</td>
+      <td>Integer</td>
+      <td>连接器将尝试执行的最大并发请求数。</td>
+    </tr>
+    <tr>
+      <td>buffered.requests.max</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">1000</td>
+      <td>Integer</td>
+      <td>每个批量请求的内存缓冲区中保留的最大请求数。</td>
+    </tr>
+    <tr>
+      <td>batch.size.max.bytes</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">5242880</td>
+      <td>Long</td>
+      <td>每个批量请求的缓冲操作在内存中的最大值。</td>
+    </tr>
+    <tr>
+      <td>buffer.time.max.ms</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">5000</td>
+      <td>Long</td>
+      <td>每个批量请求的缓冲 flush 操作的间隔。</td>
+    </tr>
+    <tr>
+      <td>record.size.max.bytes</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">10485760</td>
+      <td>Long</td>
+      <td>单个记录的最大大小（以byte为单位）。</td>
+    </tr>
+    </tbody>
+</table>    
+</div>
+
+Usage Notes
+--------
+
+* 写入 Elasticsearch 的 index 默认会是上游表 `namespace.schemaName.tableName` 对应的字符串，可以通过 pipeline 的 route 功能进行修改。
+
+* 如果写入 Elasticsearch 的 index 不存在，不会被默认创建。
+
+Data Type Mapping
+----------------
+Elasticsearch 将文档存储在 JSON 字符串中。因此数据类型映射介于 Flink 数据类型和 JSON 数据类型之间。
+<div class="wy-table-responsive">
+<table class="colwidths-auto docutils">
+    <thead>
+      <tr>
+        <th class="text-left">CDC type</th>
+        <th class="text-left">JSON type</th>
+        <th class="text-left" style="width:60%;">NOTE</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>TINYINT</td>
+      <td>NUMBER</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>SMALLINT</td>
+      <td>NUMBER</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>INT</td>
+      <td>NUMBER</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>BIGINT</td>
+      <td>NUMBER</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>FLOAT</td>
+      <td>NUMBER</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>DOUBLE</td>
+      <td>NUMBER</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>DECIMAL(p, s)</td>
+      <td>STRING</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>BOOLEAN</td>
+      <td>BOOLEAN</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>DATE</td>
+      <td>STRING</td>
+      <td>with format: date</td>
```

Comment:
Specify the date string format used here. An example of output string would be nice.